packages/insomnia-app/app/ui/helpers/sorting.js

```diff
@@ -0,0 +1,62 @@
+// @flow
+import {
+  HTTP_METHODS,
+  SortOrder,
+  SORT_CREATED_FIRST,
+  SORT_CREATED_LAST,
+  SORT_METHOD,
+  SORT_NAME_ASC,
+  SORT_NAME_DESC,
+  SORT_TYPE_ASC,
+  SORT_TYPE_DESC,
+} from '../../common/constants';
+import { request, requestGroup } from '../../models';
```

Comment:
I suggest importing the types directly, in addition to the models themselves if necessary

```suggestion
import { request, requestGroup } from '../../models';
import type { Request } from '../../models/request';
import type { RequestGroup } from '../../models/request-group';
```

Reply:
Thanks! I couldn't find a way to do this when I tried.
Also, using these `type`s in the if checks inside sorting methods does not work, so I kept those the same.

Reply:
Yep that's fine. The types aren't available at runtime so you did that check correctly.